openssl-ibmca.spec

```diff
@@ -39,7 +39,7 @@ popd
 
 %files
 %license LICENSE
-%doc ChangeLog README.md src/openssl.cnf.sample.%{_arch} src/openssl.cnf.defaultlibica src/openssl.cnf.libica src/openssl.cnf.libica-cex
+%doc ChangeLog README.md src/openssl.cnf.sample.%{_arch} src/ibmca-engine-opensslconfig
```

Comment:
Are you sure that src/ibmca-engine-opensslconfig should be specified here? Its defined as EXTRA_DIST in the makefile, so it won't be installed, and thus rpmbuild won't find it ?

Also src/openssl.cnf.sample.%{_arch} seems to be wrong...

Reply:
That is not how I understand the `%doc` directive.  It can refer to uninstalled files (like ChangeLog and README.md).  The sample config I only keep for backward compatibility.  And the script should probably not go into the documentation folder, but I have no idea where distributions want to store it.  Hopefully not under /usr/bin (or /usr/sbin)...